test-frameworks/e2e-framework/src/test/java/UserTests.java

```diff
@@ -77,4 +77,31 @@ public void listAnUser() throws IOException {
         //Assert
         Assert.assertTrue(homePO.isNameListed(user));
     }
+
+    /*
+    * Test by Team 3
+    * Objective: Check if a user is deleted as expected
+    * Scope: Delete users using the UI
+    *
+    * Scenario: Delete an user
+    *   Given the user "Alejandro-Test1" exists
+    *    When I send the command "delete"
+    *    Then I expect not to see the "Alejandro-Test1" listed in the results
+    * */
+    @Test
+    public void deleteUser() throws IOException {
+        //Arrange
+        HomePO homePO = new HomePO(driver);
+        homePO.goToHome();
+        String user = "Alejandro-Test1";
+        String command = "delete " + user;
+
```

Comment:
Congrats, It looks good!

Note: To avoid the dependency between the tests, other name should be used in this test. Something like:

```
String user = "cmpinzonh";

ApiManager.callBackendAPI("command=crear&name="+user); 
``` 

and add the deletion it in the AfterClass to delete even if something fails in the test:
```
...
ApiManager.callBackendAPI("command=borrar&name=cmpinzonh");
```